Mathlib/LinearAlgebra/Matrix/ProjectiveSpecialLinearGroup.lean

```diff
@@ -0,0 +1,27 @@
+/-
+Copyright (c) 2023 Wen Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Wen Yang
+-/
+import Mathlib.LinearAlgebra.Matrix.SpecialLinearGroup
+
+/-!
+# Projective Special Linear Group
```

Comment:
This docstring should mention the notation and which `open scoped` is needed to obtain it.